content/http-handler-abi.md

```diff
@@ -859,6 +859,31 @@ below, and the `len` would be eight.
           buf --^
 ```
 
+### `get_remote_addr`
+
+```webassembly
+;; get_remote_addr writes the client remote addr to memory if it isn't larger than `buf_limit`,
```

Comment:
why are we not calling it "source address" instead of "client remote addr"?

Reply:
term remote addr is coming from golang, but you are right it could be get_source_addr

Reply:
fixed